### PR TITLE
Added mocked localstorage.

### LINKED ISSUE #26: Support localStorage

### PR DESCRIPTION
**Pull request by andolf**
Fix #26

### CODE REVIEW

File: lib/Window.js [hunk 1]
```diff
@@ -54,6 +55,8 @@ module.exports = function Window(resp, windowObjects, innerWidth = 760, innerHei
     get: () => window
   });
 
+  Object.defineProperty(window, "localStorage", new Storage());
```
Comment (paed01) [inline lib/Window.js]: This only supplies a getter, hence the property cannot be deleted. Makes it impossible to test a scenario without localStorage.

Add the following tests:
```js
    it("can be deleted for testing purposes", () => {
      delete window.localStorage;
      expect(window.localStorage).to.be.undefined;
    });

    it("can be overwritten for testing purposes", () => {
      const mockStorage = {};
      window.localStorage = mockStorage;
      expect(window.localStorage === mockStorage).to.be.true;
    });

    it("is enumerable", () => {
      expect(Object.keys(window)).to.include("localStorage");
    });
```

**Comment by paed01** [comment]
Since local storage is persisted between navigation isn't it better to supply it as an extension, e.g. IntersectionObserver? Then you can start one and then reuse it when navigating again? @CaptainJiNX?

**Comment by CaptainJiNX** [comment]
Yes, it would probably be better to handle like that. It should not be there by default, since you want to be able to test stuff both with and without storage.

**Comment by paed01** [comment]
An unqualified guess is that both localStorage and sessionStorage behaves the same!? So the extension could be exposed as Storage. And then added to window when used.

**Comment by CaptainJiNX** [comment]
Yes they have the same interface

**Comment by andolf** [comment]
@paed01 @CaptainJiNX Check it out!

**Pull request merged**